add fontawesome via npm

creativenobu opened the pull request:
Solves issue #47

rexim commented on html/index.html:
```diff
@@ -4,13 +4,11 @@
   <body>
     <div class="header">
-      <h1>Schedule</h1>
+      <h1>Schedule  <span class="fab fa-twitch"></span></h1>
```
This is completely outside of the scope of #47. Please remove this icon.

creativenobu replied:
Ah yes, I forgot to remove. That was a check to see if it worked or not. Will update

creativenobu commented:
> Can we use only CSS part of the FontAwesome and still install it from npm?

I also don't like the JS approach, I guess I can include the FontAwesome icons using their SCSS. That way styling and logic are separate.

rexim commented:
@creativenobu I'd really appreciate if you took a look into that, yes! If it takes too much effort to implement, please, don't waste your time on it. Just post your findings in this PR and I will see what we can do about it. Research is a contribution too. Thank you so much for your time!

creativenobu commented:
So I've moved it to import FontAwesome via sass, seems like it works. I left a test case in main.scss (commented) to add FontAwesome icons in existing classes, if needed. 

I had to load the fontawesome path on the sass command, so I added SASS_FLAGS in the Makefile, that will include fontawesome scss and webfonts to the app.

I followed directions [here ](https://fontawesome.com/how-to-use/on-the-web/using-with/sass) and found the [sass cli ](https://sass-lang.com/documentation/cli/dart-sass) that had that option (fortunately?) 😄

rexim commented:
@creativenobu this is waay better! Thank you so much for your contribution! :+1:
